.pre-commit-config.yaml

```diff
@@ -93,7 +93,7 @@ repos:
 
             # requirements-test.txt
             coverage>=5.0.1,
-            dask>=2.29.0,
+            dask[dask-expr,dataframe]>=2020.12.0,
```

Comment:
This is actually failing the `pre-commit` locally on my end... did you commit this with the pre-commit enabled @gliptak?

Reply:
no, I modified it on the GUI. the error message refers to Unicode which is confusing ...

Reply:
reverted this update. would recommend addressing in separate PR